oak_functions/loader/src/tests.rs

```diff
@@ -56,7 +56,7 @@ async fn test_server_without_policy() {
 async fn test_valid_policy() {
     // Policy values are large enough to allow successful serving of the request, and responding
     // with the actual response from the Wasm module.
-    let constant_processing_time = Duration::from_millis(100);
+    let constant_processing_time = Duration::from_millis(200);
```

Comment:
@rbehjati FYI I had to bump this up because it was occasionally exceeding it on my machine. I think it's unrelated to my change, but just a heads-up.

Reply:
SGTM. Thanks.